packages/backend/src/modules/tracked-txs/utils/findConfigurationsToSync.ts

```diff
@@ -0,0 +1,54 @@
+import { assert } from '@l2beat/backend-tools'
+import { notUndefined, UnixTime } from '@l2beat/shared-pure'
+
+import { TrackedTxsConfigRecord } from '../repositories/TrackedTxsConfigsRepository'
+import { TrackedTxConfigEntry } from '../types/TrackedTxsConfig'
+import { isTimestampInRange } from './isTimestampInRange'
+
+export function findConfigurationsToSync(
```

Comment:
Please provide a comment with the simple A,B,C example of until timestamp being used